core/pages/Category.js

```diff
@@ -92,9 +94,11 @@ export default {
           reject(err)
         }).then((attrs) => {
           store.dispatch('category/single', { key: 'slug', value: route.params.slug }).then((parentCategory) => {
```

Comment:
We should move all these vuex calls to vuex action and separate it from component properties itself like it’s currrntly (almost) achieved in Product.js

I mean: we should have as thin ui components as we can as it gives us a possibility to better separate Ui and business logic in the next rrleases